pyproject.toml

```diff
@@ -22,6 +22,7 @@ pylev = "^1.3"
 
 # The typing module is not in the stdlib in Python 2.7 and 3.4
 typing = { version = "^3.6", python = "~2.7 || ~3.4" }
+typing-extensions = { version = "^3.6", python = "~3.5 || ~3.6" }
```

Comment:
Do we actually need `~3.6`?

Also, shouldn't it be `>=3.5.0,<3.5.4`?

Reply:
You're right, we don't. I was thinking that typing-extensions provides some addtional types that are missing on 3.6, but none of them are in use in clikit.